models/history/salesforce__contact_daily_history.sql

```diff
@@ -0,0 +1,87 @@
+{{ config(enabled=var('salesforce__contact_history_enabled', False)) }}
+
+{{
+    config(
+        materialized='incremental',
+        partition_by = {'field': 'date_day', 'data_type': 'date'}
+            if target.type not in ['spark', 'databricks'] else ['date_day'],
+        unique_key='contact_day_id',
+        incremental_strategy = 'merge' if target.type not in ('snowflake', 'postgres', 'redshift') else 'delete+insert',
+        file_format = 'delta'
+    )
+}}
+
+with spine as (
+
+    {% if execute %}
+    {% if not var('contact_history_first_date', None) or not var('contact_history_last_date', None) %}
```

Comment:
Resolved in person, but for context sake: The earliest `_fivetran_start` date should have historical records preceding it. The earliest `created_at` date doesn't necessarily have historical records, as history mode might not have been available by that point. That would lead to numerous null records.